omod/package.json

```diff
@@ -3,8 +3,7 @@
   "version": "0.0.1",
   "description": "Mirebalais - Apppointment Scheduling Ui",
   "dependencies": {},
-  "devDependencies": {
-  },
+  "devDependencies": {},
```

Comment:
cc @dkayiwa: Whwn comparing this code here from master https://github.com/openmrs/openmrs-module-appointmentschedulingui/blob/master/omod/package.json#L6. and this same line in my commit,they look same, the above change was rectified thanks